### ISSUE TITLE
Cypress detected that you returned a promise from a command while also invoking one or more cy commands in that promise

### ISSUE THREAD

**Issue by jean9696** [archfz/cypress-terminal-report#35]
Hi, I randomly get this error while running my tests in CI.

I cannot reproduce it in local and I don't understand what is happening. 

<img width="413" alt="Capture d’écran 2020-06-11 à 17 19 12" src="https://user-images.githubusercontent.com/13215843/84404955-c64e7300-ac07-11ea-8ee4-731f9a2ce5bf.png">

https://github.com/archfz/cypress-terminal-report/blob/master/src/installLogsCollector.js#L74

```CypressError: The following error originated from your test code, not from Cypress.

  > Cypress detected that you returned a promise from a command while also invoking one or more cy commands in that promise.

The command that returned the promise was:

  > `cy.wrap()`

The cy command you invoked inside the promise was:

  > `cy.task()`

Because Cypress commands are already promise-like, you don't need to wrap them or return your own promise.

Cypress will resolve your command with whatever the final Cypress command yields.

The reason this is an error instead of a warning is because Cypress internally queues commands serially whereas Promises execute as soon as they are invoked. Attempting to reconcile this would prevent Cypress from ever resolving.

When Cypress detects uncaught errors originating from your test code it will automatically fail the current test.

Because this error occurred during a `after each` hook we are skipping all of the remaining tests.```

**Comment by archfz** [comment]
@jean9696 Are you using cypress 4.8? If yes the newest version is not yet incompatible with this plugin atm. You can downgrade cypress yo 4.7 or maybe help port it for the newer version :).

**Comment by archfz** [comment]
Released in 1.4.0
